### ISSUE TITLE
Использование кода

### ISSUE THREAD

**Issue by muslii** [antimalware/manul#117]
Добрый день. Меня интересует вопрос, могу ли я использовать некоторый код  из Manul в своем  не коммерческом проекте. Так как мне немного не подходит он  и хотелось бы переделать функционал под себя. Заранее благодарен за ответ.

**Comment by peter-volkov** [comment]
Можете, согласно BSD License

> Redistribution and use in source and binary forms, with or without modification,
> are permitted provided that the following conditions are met:
> 
>   Redistributions of source code must retain the above copyright notice, this
>   list of conditions and the following disclaimer.
> 
>   Redistributions in binary form must reproduce the above copyright notice, this
>   list of conditions and the following disclaimer in the documentation and/or
>   other materials provided with the distribution.
>

**Comment by muslii** [comment]
peter-volkov, то есть мне  нужно указать данных текст в каждом файле ?

**Comment by peter-volkov** [comment]
Нет, достаточно в общей лицензии, если она есть в проекте. Вообще не следует заморачиваться - берите код и используйте там, где считаете уместным
